pkg/server/server_sql.go

```diff
@@ -610,6 +610,23 @@ func (s *sqlServer) start(
 			// present situation).
 			DistSQLMode: sessiondata.DistSQLOff,
 		})
+	{
+		// The server's internalExecutor might be used as a side effect of
+		// migrations, so disable distribution for the same reason as above until
+		// all migrations have been run.
+		sessionDataCopy, sessionDataWasNil := s.internalExecutor.GetSessionData()
```

Comment:
Can the results of `GetSessionData()` vary at all? If not, let's just assert that they are what they should be and don't bother keeping track of the old settings. In fact, I wouldn't even call `GetSessionData()`.

Reply:
I don't think so since this is all in the startup path. Removed `GetSessionData`.